_layouts/default.html

```diff
@@ -1,9 +1,9 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
Is the `shrink-to-fit=no` in reference to the Mac/iOS issues in Version 9.x? If so, is that still needed as a fix? Genuinely asking as I haven't done many cross-browser or cross-OS testing in this site. I don't know if [this article](https://www.scottohara.me/blog/2018/12/11/shrink-to-fit.html) has merit.

How would this affect the layout of the site across different browsers? My goal is to have the swag list as simple and universal as possible.